.gitignore

```diff
@@ -42,6 +42,7 @@ gradle.properties
 
 # Cyface Temporary Files
 **/uploadFolder
+**/file-uploads
```

Comment:
In some places we use the one, in the other the other folder name (injected via parameters) so I just added both to gitignore.